django/contrib/postgres/forms/jsonb.py

```diff
@@ -0,0 +1,29 @@
+import json
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+__all__ = ['JSONField']
+
+
+class JSONField(forms.CharField):
+    default_error_messages = {
+        'invalid': _("'%(value)s' value must be valid JSON."),
+    }
+
+    def __init__(self, **kwargs):
+        kwargs.setdefault('widget', forms.Textarea)
+        super(JSONField, self).__init__(**kwargs)
+
+    def to_python(self, value):
+        try:
+            return json.loads(value)
+        except ValueError:
```

Comment:
Shouldn't we only be attempting to `json.loads(value)` if value is a string object?

From https://docs.djangoproject.com/en/1.8/howto/custom-model-fields/#converting-values-to-python-objects:

>As a general rule, to_python() should deal gracefully with any of the following arguments:
>  * An instance of the correct type (e.g., Hand in our ongoing example).
>  * A string
>  *None (if the field allows null=True)